packages/components/src/SelectBox/SelectBox.tsx

```diff
@@ -0,0 +1,280 @@
+      color: theme.colors.text.primary,
+      backgroundColor: theme.colors.background.primary,
+      boxShadow: 0,
+      borderRadius: theme.border.radii[1],
+      borderColor: error
+        ? theme.colors.feedback.error
+        : state.isDisabled
+        ? theme.colors.shades.grey204
+        : state.isFocused
+        ? theme.colors.shades.grey42
+        : theme.colors.shades.grey143,
+      ':hover': {
+        borderColor: error
+          ? theme.colors.feedback.error
+          : state.isDisabled
+          ? theme.colors.shades.grey204
+          : state.isFocused
+          ? theme.colors.shades.grey42
+          : theme.colors.shades.grey143,
+      },
+    };
+  },
+  placeholder: (provided: any) => ({
+    ...provided,
+    opacity: '0.6',
+  }),
+  indicatorSeparator: (provided: any, state: any) => ({
+    ...provided,
+    margin: 0,
+    backgroundColor: error
+      ? theme.colors.feedback.error
+      : state.isDisabled
+      ? theme.colors.shades.grey204
+      : state.isFocused
+      ? theme.colors.shades.grey42
+      : theme.colors.shades.grey143,
+  }),
+  multiValue: (provided: any) => ({
+    ...provided,
+    backgroundColor: theme.colors.background.secondary,
+    borderRadius: '30px',
+  }),
+  multiValueRemove: (provided: any) => ({
+    ...provided,
+    ':hover': {
+      background: 'inherit',
+      borderRadius: 'inherit',
+      cursor: 'pointer',
+    },
+  }),
+  valueContainer: (provided: any) => ({ ...provided, cursor: 'text' }),
+  clearIndicator: (provided: any) => ({ ...provided, cursor: 'pointer' }),
+  dropdownIndicator: (provided: any) => ({ ...provided, cursor: 'pointer' }),
+  option: (provided: any, state: any) => ({
+    ...provided,
+    backgroundColor: state.isDisabled
+      ? null
+      : state.isSelected
+      ? theme.colors.shades.grey232
+      : state.isFocused
+      ? theme.colors.shades.grey244
+      : null,
+    color: state.isDisabled ? theme.colors.shades.grey232 : null,
+    cursor: state.isDisabled ? 'not-allowed' : 'default',
+    ':active': {
+      backgroundColor:
+        !state.isDisabled &&
+        (state.isSelected
+          ? theme.colors.shades.grey232
+          : state.isFocused
+          ? theme.colors.shades.grey244
+          : null),
+    },
+  }),
+  noOptionsMessage: (provided: any) => ({ ...provided, textAlign: 'left' }),
+  group: (provided: any) => ({
+    ...provided,
+    ':not(:last-child)': {
+      borderBottom: '1px solid',
+      borderBottomColor: theme.colors.shades.grey232,
+      marginBottom: theme.space[2],
+    },
+  }),
+});
+
+const ClearIndicator = (props: any) => {
+  const theme = useTheme() as Theme;
+
+  return (
+    <components.ClearIndicator {...props}>
+      <Icon
+        component={MaterialClear}
+        width="20px"
+        height="20px"
+        fill={theme.colors.shades.grey143}
+      />
+    </components.ClearIndicator>
+  );
+};
+
+const DropdownIndicator = (error: boolean, theme: Theme & DefaultTheme) => (
```

Comment:
Kannst du die Funktion hier bitte in `createDropdownIndicator` oder so umbenennen? Die gibt ja nicht direkt eine React-Komponente zurück und braucht deshalb andere Properties.